XandO.java

```diff
@@ -1,5 +1,104 @@
+import java.util.*;
+
 class PlayGame {
+    private static Scanner in = new Scanner(System.in);
     public static void main (String args[]) {
         XandO_Game xg = new XandO_Game();
+        startGame(xg);
+    }
+
+    // TODO: Move this into api class better better encapsulation.
+    // TODO: Break below function into more smaller logical components for simplicity
+    // TODO: Currently it is single time play per run, and can be played on single machine. Change that in future.
+    private static void startGame(XandO_Game xg) {
+        initialInstructions(xg);
+        int player = 1;
+        char move = selectPlayerOneChar();
+        
+        do {
+            try {
+                System.out.println("Player "+player+" turn to play");
+                
+                int row = in.nextInt();
+                if (row == -1) {
+                    System.exit(0);
```

Comment:
Exiting the program for incorrect input may ruin the fun of the play. If I were to play a game I would expect a message to enter correct input. Your call.